projects/core/src/lib/operators/dialogs/rasterization/rasterization.component.html

```diff
@@ -0,0 +1,79 @@
+<geoengine-sidenav-header>Rasterization</geoengine-sidenav-header>
+
+<geoengine-dialog-help>
+    <p>
+        TODO This operator filters the given point layer using the given polygon layer. The resulting layer contains only (multi) points
+        that are contained by at least one polygon spatially and temporally.
+    </p>
+</geoengine-dialog-help>
+
+<form [formGroup]="form" (ngSubmit)="$event.preventDefault(); add()">
+    <div class="container">
+        <geoengine-layer-selection [types]="inputTypes" formControlName="layer"></geoengine-layer-selection>
+        <geoengine-dialog-section-heading title="Config" subtitle="Specify the rasterization parameters">
+        </geoengine-dialog-section-heading>
+        <div formGroupName="rasterization">
+            <mat-tab-group
+                color="primary"
+                disablePagination
+                [selectedIndex]="selected.value"
+                (selectedIndexChange)="selected.setValue($event)"
+            >
+                <mat-tab label="Grid">
+                    <ng-container *ngTemplateOutlet="gridTemplate"></ng-container>
+                </mat-tab>
+                <mat-tab label="Density">
+                    <ng-container *ngTemplateOutlet="densityTemplate"></ng-container>
+                </mat-tab>
+            </mat-tab-group>
+
+            <ng-template #gridTemplate>
+                <label id="radio-group-label" class="labelRadioGroup">Size mode:</label>
+                <mat-radio-group color="primary" formControlName="gridSizeMode" aria-labelledby="radio-group-label">
+                    <mat-radio-button value="fixed">Fixed</mat-radio-button>
+                    <mat-radio-button value="relative">Relative</mat-radio-button>
+                </mat-radio-group>
+                <div formGroupName="resolution" class="groupInRow">
+                    <mat-form-field appearance="fill">
+                        <mat-label>Resolution (x)</mat-label>
+                        <input matInput type="number" formControlName="resX" />
+                    </mat-form-field>
+                    <mat-form-field appearance="fill">
+                        <mat-label>Resolution (y)</mat-label>
+                        <input matInput type="number" formControlName="resY" />
```

Comment:
it shouldn't be allowed to have zero or negative resolutions

Reply:
da68e10